tools/sklearn/discriminant.xml

```diff
@@ -32,6 +32,9 @@ with open(input_json_path, "r") as param_handler:
 #if $selected_tasks.selected_task == "load":
 
 classifier_object = io.load("$infile_model")
+module_name = classifier_object.__module__
+class_name = classifier_object.__class__.__name__
+SafePickler.find_class(module_name, class_name)
```

Comment:
I doubt the previous method still works here. 1) the code in the model file is already executed in the `io.load`. 2) it seems that the `SafePickerler` only checks the top level of the object.